packages/ui/src/Link.tsx

```diff
@@ -40,8 +41,8 @@ type LinkProps = IconProps & {
  * - Link can be used as a stand-alone component with right chevron icon.
  */
 export const Link: FC<LinkProps> = ({
-  size = 'regular',
-  Icon,
+  size = 'normal',
+  icon,
```

Comment:
Shouldn't we mirror Figma, as for naming? I would personally revert to the previous naming.

Reply:
I used `normal` and `small` because we have it everywhere - icon sizes, typography, ...